src/components/VmDialog/index.js

```diff
@@ -316,6 +324,34 @@ class VmDialog extends React.Component {
     }
   }
 
+  checkTimeZone ({ osType }) {
+    let timeZone = null
+    const template = this.getTemplate()
+    if (template) {
+      if (template.getIn(['timeZone', 'name'])) {
+        timeZone = timeZone || template.get('timeZone').toJS()
+        const isWindowsTimeZone = timezones.find(timezone => timezone.id === timeZone.name)
+        const isWindowsVm = isWindows(osType)
+
+        if (isWindowsVm && !isWindowsTimeZone) {
```

Comment:
What if OS is non-windows type (isWindowsVm=false) and timezone is set to windows type? Don't you need to check both like in .

Reply:
indeed, need to work both ways

Reply:
It is already works in both ways
https://github.com/oVirt/ovirt-web-ui/blob/71335fa5c1c0e457aae11e6b2235dd8e828e34b9/src/components/VmDialog/index.js#L336-L346